doc/tools/parse_latex_log.py

```diff
@@ -233,53 +199,68 @@ def __init__(self, log_path, src_dir, verbose=False):
         self.src_dir = src_dir
         self.verbose = verbose
 
-    def _read_tex_error(self, line):
+    def _read_tex_error(self):
         line_no = parse_line_number(self._current_issue)
         self._issues['issues'].append({
-            'severity': SEVERITY_ERROR,
+            'level': SEVERITY_ERROR,
             'type': self._type,
-            'locations': [{
-                'file': self._current_tex_file,
-                'line': line_no}],
+            'path': self._current_tex_file,
+            'line': {"start": line_no, "end": line_no},
             'message': self._current_issue.strip()})
 
-    def _read_warning(self, line):
-        warn = parse_warning(self._current_issue, self.src_dir)
-        if self._current_issue.strip() not in ISSUES_TO_SKIP:
-            if warn is not None:
-                self._issues['issues'].append(warn)
-            else:
-                line_no = parse_on_input_line(self._current_issue)
-                self._issues['issues'].append({
-                    'severity': SEVERITY_WARNING,
-                    'type': self._type,
-                    'locations': [{
-                        'file': self._current_tex_file,
-                        'line': line_no}],
-                    'message': self._current_issue.strip()})
-
-    def _read_error(self, line):
-        if self._current_issue not in ISSUES_TO_SKIP:
+    def _read_warning(self):
+        warns = []
+        m1 = LABEL_MULTIPLY_DEFINED_WARN.match(self._current_issue)
+        m2 = HYPER_UNDEFINED_WARN.match(self._current_issue)
+        if m1 is not None:
```

Comment:
And so presumably this is where we would match other LaTeX warnings if we wanted.